src/lib/eddsa.c

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2017, [Ribose Inc](https://www.ribose.com).
+ * All rights reserved.
+ *
+ * This code is originally derived from software contributed to
+ * The NetBSD Foundation by Alistair Crooks (agc@netbsd.org), and
```

Comment:
I'm not sure that this code from NetBSD. I think we should remove this line from this file, shouldn't we?